meetings/2019-03-f2f.md

```diff
@@ -0,0 +1,247 @@
+# AMP F2F 2019-03-27/28 in London
+
+**Agenda:** https://github.com/ampproject/meta-ac/issues/22
+
+## Day 1
+
+#### Attendees:
+
+* Terence Eden @edent
+* Tobie Langel @tobie
+* Charles Vazac @cvazac
+* Joe Alicata @wirelessjoe
+* Levi Durfee @levidurfee
+* Léonie Watson @ljwatson
+
+### What is the TSC
+
+* Technical Steering Committee.
+* Inspired by NodeJS style terminology.
+* Responsible for handling project day to day activity.
+* Roadmap, direction. Project management / tech lead distributed throughout a group.
+* TSC delegate to working groups.
+* 3 out of 7 members are from Google for now.
+* Goal is that no more than 1/3 of members are from the same company.
+
+### Scope and Role of the AC
+
+* AC is Advisory to the TSC.
+* Advice is non-binding.
+* AC can provide opinions / advice / findings.
+* Once we have a Foundation, we will have a board
+  * AC should be represented on the Board
+  * TSC should be represented on the Board
+  * The Board will be respnsible for finance, IP, and governance
```

Comment:
The Board will be responsible for finance, IP, and governance

Reply:
```suggestion
  * The Board will be responsible for finance, IP, and governance
```